drivers/linstorvolumemanager.py

```diff
@@ -2598,7 +2669,7 @@ def _get_error_str(cls, result):
 
     @classmethod
     def _create_linstor_instance(
-        cls, uri, keep_uri_unmodified=False, attempt_count=30
+        cls, uri, keep_uri_unmodified=False, keep_cache_on_error=False, attempt_count=30
```

Comment:
I guess you can remove this new `keep_cache_on_error` param if not used. Don't see a good use case for now.

Reply:
removed in [cbdf27c](https://github.com/xcp-ng/sm/pull/83/commits/cbdf27cdd10fc2c00fcfbe003607524b8fa9af7c).